topics/cli.md

```diff
@@ -215,6 +215,10 @@ This topic covers the different aspects of `valkey-cli`, starting from the simpl
 : Sample keys looking for hot keys.
   only works when maxmemory-policy is `*lfu`.
 
+**`--hotkeys-count`**
+: Number of Sample keys looking for hot keys.
+  only works when maxmemory-policy is `*lfu`.
```

Comment:
Let's use the same text as in `valkey-cli --help`

```suggestion
  This is equivalent to --hotkeys-count 16.

**`--hotkeys-count`** _n_
: Sample keys looking for the *n* most hot keys.
  Only works when maxmemory-policy is `*lfu`.
```

Reply:
You can just click the button `Sign off and commit suggestion`. I think it is identical.